src/routing_node.rs

```diff
@@ -64,6 +65,7 @@ pub struct RoutingNode<F: Facade> {
     accepting_on: Option<u16>,
     next_message_id: MessageId,
     bootstrap_node_id: Option<DhtId>,
+    filter: LruCache<types::FilterType, ()>,
```

Comment:
I think this should be a message_filter type as opposed to an lru_cache ? (i.e no values)